lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.h

```diff
@@ -80,6 +80,11 @@ class TypeSystemSwiftTypeRef : public TypeSystemSwift {
   CompilerType GetGenericArgumentType(lldb::opaque_compiler_type_t type,
                                       size_t idx) override;
 
+  /// Returns the list of DependentGenericParamTypes (depth, index pairs) that a
+  /// type has, if any.
```

Comment:
Can you check if there is a term of art for this? I.e., how is this different from a swift::GenericSignature?

Reply:
Turns out this isn't needed, this is equivalent to generic params of a generic signature.